moto/s3/responses.py

```diff
@@ -466,6 +466,13 @@ def _bucket_response_get(self, bucket_name, querystring):
                     is_truncated="false",
                 ),
             )
+        elif "encryption" in querystring:
+            bucket = self.backend.get_bucket(bucket_name)
```

Comment:
Would you mind adding the logic to a `the get_bucket_encryption` in models.py? Makes this a bit more readable.
```
encryption = self.backend.get_bucket_encryption(..)
if not encryption..
```